Sources/MongoSwift/AsyncAwait/MongoDatabase+AsyncAwait.swift

```diff
@@ -27,6 +27,8 @@ extension MongoDatabase {
      *
      * - Throws:
      *   - `MongoError.CommandError` if an error occurs that prevents the command from executing.
+     *     If a collection with this name already exists, a `MongoError.CommandError` with code `48`
+     *     will be thrown.
```

Comment:
I think it might be good to include the `NamespaceExists` codename in the docstrings too.